CodeApp/Containers/SourceControlContainer.swift

```diff
@@ -72,8 +72,21 @@ struct SourceControlContainer: View {
         return try await withCheckedThrowingContinuation { continuation in
             serviceProvider.push(
                 error: {
-                    App.notificationManager.showErrorMessage(
-                        $0.localizedDescription)
+                    if $0.code == GitErrorCode.GIT_EAUTH.rawValue {
+                        App.notificationManager.postActionNotification(
+                            title:
+                                "Authentication failed: You might need to configure your git credentials.",
```

Comment:
Missing localisation

Reply:
This will not be shown if a wrong credential is configured:
<img width="506" alt="image" src="https://user-images.githubusercontent.com/38398443/202649715-05bb1705-8e25-4855-a21c-e147c3989170.png">

Changes need to be made in SwiftGit2 to return the correct error code.